src/Compilers/Core/Portable/Symbols/LanguageNames.cs

```diff
@@ -20,6 +20,9 @@ public static class LanguageNames
         /// <summary>
         /// The common name used for the F# language.
         /// </summary>
+        /// <remarks>
+        /// This value is not currently supported by Roslyn.
```

Comment:
```suggestion
        /// F# is not a supported compile target for the Roslyn compiler.
```

@CyrusNajmabadi how about that?

Reply:
Sounds good :)